samples/event-api-handlers/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM amazoncorretto:21-alpine3.18@sha256:e937a2668732ad0e138180f40e26586b4ba39282271889466ce6693410b77f2c
+FROM amazoncorretto:17-alpine3.18@sha256:e937a2668732ad0e138180f40e26586b4ba39282271889466ce6693410b77f2c
```

Comment:
Does the different java version have the same SHA? Seems weird, I don't think you can blindly replace this with a different number

Reply:
Yes updated the SHA.